### DISCUSSION TITLE
What does this output mean: ll+, now 31 ll+, now 32 ll+, now 33 ll+, now 34 ll+, now 35 ll+, now 36 ll+, now 37 ll+, now 38

### DISCUSSION THREAD

**Discussion by charlesfayal** [bemasher/rtlamr#286] [Q&A]
I see this occasionally as an output:
ll+, now 31
ll+, now 32
ll+, now 33
ll+, now 34
ll+, now 35
ll+, now 36
ll+, now 37
ll+, now 38
..

**Comment by electronhauler** [comment]
I'm getting the same thing
set freq 912600155
set sample rate 2359296
set gain mode 0
ll+, now 1
ll-, now 0
ll+, now 1
ll-, now 0
ll+, now 1
ll-, now 0
ll+, now 1
ll+, now 2
ll+, now 3
ll-, now 0
ll+, now 1
ll+, now 2
ll+, now 3
ll+, now 4
ll+, now 5
ll-, now 0
ll+, now 1
ll-, now 0
ll+, now 1
ll+, now 2
ll+, now 3
ll+, now 4
ll+, now 5
ll-, now 0
ll+, now 1
ll-, now 0
ll+, now 1
ll-, now 0
